impl/src/main/java/org/jboss/weld/util/reflection/Reflections.java

```diff
@@ -493,4 +499,36 @@ public static boolean isDefault(Method method) {
         return ((method.getModifiers() & (Modifier.ABSTRACT | Modifier.PUBLIC | Modifier.STATIC)) == Modifier.PUBLIC)
                 && method.getDeclaringClass().isInterface();
     }
+
+    public static boolean hasBeanDefiningAnnotation(Class<?> clazz, Set<Class<? extends Annotation>> initialBeanDefiningAnnotations) {
```

Comment:
Wrong `Reflections` class ;-) It should be `org.jboss.weld.environment.util.Reflections`.